WORKSPACE

```diff
@@ -323,10 +323,10 @@ container_pull(
 # instead of index.docker.io
 container_pull(
     name = "fedora_sriov_lane",
-    digest = "sha256:2d332d28863d0e415d58e335e836bd4f8a8c714e7a9d1f8f87418ef3db7c0afb",
+    digest = "sha256:48911736a83b141cb001daeec4148bfe51076bb9701fe4b2f71fb57a8f729ca2",
     registry = "quay.io",
-    repository = "kubevirtci/fedora-sriov-testing",
-    #tag = "32",
+    repository = "oshoval/fedora-sriov-lane-container-disk-test",
```

Comment:
Once the PR is approved, will push to an official repo

Reply:
which PR? this one?

Reply:
https://github.com/kubevirt/kubevirtci/pull/556 need to be lgtm approve + merge
and then the current PR https://github.com/kubevirt/kubevirt/pull/5135 need to be lgtm + approve + hold
and then i will change this snippet to an image which will be build due to https://github.com/kubevirt/kubevirtci/pull/556 
and pushed to quay.io/kubevirtci, and remove the hold